models/src/main/protobuf/CasperMessage.proto

```diff
@@ -229,6 +234,7 @@ message PeekProto {
 message BondProto {
   bytes validator = 1;
   int64 stake     = 2;
+  bool  deactivated  = 3;
```

Comment:
If we are not using this right now, maybe it makes sense to postpone adding it later. It will not cause incompatibility in any way.